coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
In future projects please make sure to add the following line to your `.gitignore` file (or copy everything from the [official Github .gitignore for Ruby](https://github.com/github/gitignore/blob/master/Ruby.gitignore)):
```
/coverage/
```
This will prevent accidentally checking in these coverage files, which are temporary in nature and do not need to be included in git.